lib/cylc/review_dao.py

```diff
@@ -539,7 +558,11 @@ def get_suite_cycles_summary(
             " sum(" + states_stmt["success"] + ") AS n_success,"
             " sum(" + states_stmt["fail"] + ") AS n_fail"
             " FROM task_states" +
-            " GROUP BY cycle")
+            " GROUP BY cycle" +
+            " HAVING sum(" + states_stmt["active"] + ")>0" +
+            " OR sum(" + states_stmt["success"] + ")>0" +
+            " OR sum(" + states_stmt["fail"] + ")>0"
```

Comment:
I like the `HAVING`, much nicer than filtering it in the Python layer.

Since we've already computed the counts of active/success/fail task groups above, we might as well reuse them here. I think this does the same thing, but only performs the `sum()` operations once:

```suggestion
            " HAVING n_active > 0" +
            " OR n_success > 0" +
            " OR n_fail >0"
```